CONTRIBUTING.md

```diff
@@ -0,0 +1,169 @@
+# Contributing to CopilotKit
+
+⭐ Thank you for your interest in contributing!!
+
+Here’s how you can contribute to this repository
+
+## How can I contribute?
+
+Ready to contribute but seeking guidance, we have several avenues to assist you. Explore the upcoming segment for clarity on the kind of contributions we appreciate and how to jump in. Reach out to us directly on [Discord](https://discord.gg/6dffbvGU3D) for immediate assistance! Alternatively, you're welcome to raise an issue and one of our dedicated maintainers will promptly steer you in the right direction!
+
+## Found a bug?
+
+If you find a bug in the source code, you can help us by [submitting an issue](https://github.com/CopilotKit/CopilotKit/issues/new?assignees=&labels=bug&projects=&template=bug_report.yaml) to our GitHub Repository. Even better, you can submit a Pull Request with a fix.
+
+## Missing a feature?
+
+So, you've got an awesome feature in mind? Throw it over to us by [creating an issue](https://github.com/CopilotKit/CopilotKit/issues/new?assignees=&labels=feature-request&projects=&template=feature_request.yaml) on our GitHub Repo.
+
+If you don't feel ready to make a code contribution yet, no problem! You can also check out the [documentation issues](https://github.com/CopilotKit/CopilotKit/issues?q=is%3Aopen+is%3Aissue+label%3Adocumentation).
+
+# How do I make a code contribution?
```

Comment:
isn't this is the answer to previous question How can I contribute you can redesign this part a bit i think